### PR TITLE
Include version number in firstrunwizard

### PR DESCRIPTION
**Pull request by szaimen**
Fix https://github.com/nextcloud/firstrunwizard/issues/82

| Before | After |
|---|---|
| image | image |

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Comment by juliushaertl** [comment]
Looks good code wise but please have a quick look @nextcloud/designers :)

File: templates/page.values.php [hunk 1]
```diff
@@ -50,5 +50,8 @@
 		</ul>
 
 		<p class="details-link"><a href="<?php p($theme->getBaseUrl()); ?>" target="_blank" rel="noreferrer noopener"><?php p($l->t('Learn more about %s', $theme->getName())); ?></a></p>
+		<div style="text-align: center;">
+			<h3><?php p($l->t('This Nextcloud is on version') . ' ' . \OC_Util::getVersionString()); ?></h3>
```
Comment (juliushaertl) [inline templates/page.values.php:54]: Probably better this way to make sure the version is properly embedded in translated text:

```suggestion
			<h3><?php p($l->t('This Nextcloud is on version %s', [ \OC_Util::getVersionString() ])); ?></h3>
```

Comment (szaimen) [reply templates/page.values.php]: done

File: templates/page.values.php [hunk 2]
```diff
@@ -50,5 +50,8 @@
 		</ul>
 
 		<p class="details-link"><a href="<?php p($theme->getBaseUrl()); ?>" target="_blank" rel="noreferrer noopener"><?php p($l->t('Learn more about %s', $theme->getName())); ?></a></p>
+		<div style="text-align: center;">
+			<h3><?php p($l->t('This Nextcloud is on version %s', [ \OC_Util::getVersionString() ])); ?></h3>
```
Comment (tcitworld) [inline templates/page.values.php:54]: As the rest of the steps use the theme name a lot I would suggest something more like :
`$theme->getName()` is using Nextcloud version %s

Comment (szaimen) [reply templates/page.values.php:54]: Hm... I would actually rather do that in a follow up to this PR...
@juliushaertl is that fine?

Comment (jancborchardt) [reply templates/page.values.php:54]: If we do show the version here, it should be rather short and inconspicuous. :) And it’s not a h3, a `<p>` with `color: var(--text-maxcontrast);` would be better. And shorter wording, simply `Nextcloud 22.0.0`.

Comment (szaimen) [reply templates/page.values.php]: done
This is how it looks now:
image

**Review by marcoambrosini — APPROVED** [review]
